packages/flutter/lib/src/material/divider.dart

```diff
@@ -96,25 +116,30 @@ class Divider extends StatelessWidget {
   /// )
   /// ```
   /// {@end-tool}
-  static BorderSide createBorderSide(BuildContext context, { Color color, double width = 0.0 }) {
-    assert(width != null);
+  static BorderSide createBorderSide(BuildContext context, { Color color, double width }) {
     return BorderSide(
-      color: color ?? Theme.of(context).dividerColor,
-      width: width,
+      color: color ?? DividerTheme.of(context).color ?? Theme.of(context).dividerColor,
```

Comment:
is Theme.of(context).dividerColor always defined?

Reply:
yep, it always has a value.